src/renderer/src/routes/pages/Settings/index.tsx

```diff
@@ -0,0 +1,141 @@
+import { useState } from 'react'
+import { useTranslation } from 'react-i18next'
+import { BsCash } from 'react-icons/bs'
+import { MdOutlineLanguage, MdOutlineListAlt, MdOutlineLock, MdOutlineSave } from 'react-icons/md'
+import { RiShapesLine } from 'react-icons/ri'
+import { Tb3DCubeSphere, TbReload } from 'react-icons/tb'
+import { MainLayout } from '@renderer/layouts/Main'
+
+import { SettingsSidebarLink } from '../../../components/Sidebar/SettingsSidebarLink'
+
+enum SettingsSidebarOption {
+  PERSONALISATION = 1,
+  SECURITY = 2,
+}
+
+enum SettingsPersonalisationOption {
+  NETWORK_CONFIGURATION = 1,
+  CURRENCY = 2,
+  LANGUAGE = 3,
+  THEME = 4,
+  RELEASE_NOTES = 5,
+}
+
+enum SettingsSecurityOption {
+  ENCRYPT_KEY = 1,
+  RECOVER_WALLET = 2,
+  BACKUP_WALLET = 3,
+}
```

Comment:
Add the prefix "E" to the name of the enumeration, because it is more descriptive when using that it is an enum, for example, ESettingsSidebarOption, etc